### PR TITLE
adding scalaz depedency and problems for beginners

### PR DESCRIPTION
**Pull request by afsalthaj**

### CODE REVIEW

**Comment by afsalthaj** [comment]
@sujeshchirackkal ... see if its ok? Currently updating just 1 problem

**Comment by afsalthaj** [comment]
@sujeshchirackkal ...yes il update today

**Comment by afsalthaj** [comment]
@sujeshchirackkal  adding a few more questions while I squashed my previous commits.

File: build.sbt [hunk 1]
```diff
@@ -1 +1,3 @@
-scalaVersion := "2.11.7"
\ No newline at end of file
+scalaVersion := "2.11.7"
+
+libraryDependencies := Seq ("org.scalaz"  % "scalaz-core_2.9.2"   % "6.0.4")
```
Comment (shyam334) [inline build.sbt]: That looks like an old version of `scalaz` and why `scala` 2.9, wouldn't it be binary incompatible?
Maybe `libraryDependencies += "org.scalaz" %% "scalaz-core" % "7.1.5"` is what you're after.

**Comment by shyam334** [comment]
Good work. Keep em coming. :+1:

File: build.sbt (hunk 1 above)
Comment (afsalthaj) [inline build.sbt]: Yea. That's true. Thanks for bringing it up. It compiled but we may not be able to import it once we start using it. Il change

**Comment by sujeshchirackkal** [comment]
:+1: Please merge

**Comment by afsalthaj** [comment]
Cant see merge option....

**Comment by sujeshchirackkal** [comment]
Can't you force merge from your branch to master ?

**Comment by afsalthaj** [comment]
Not sure. It is said here that only those who have write access can merge pull requests

**Comment by sujeshchirackkal** [comment]
was missing write access. Have provided now. Shyam has already merged.

On Wed, Dec 2, 2015 at 7:50 PM, afsalthaj notifications@github.com wrote:

> Not sure. It is said here that only those who have write access can merge
> pull requests
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/DigitalEarns/ScalaExercises/pull/1#issuecomment-161223495
> .

## 

Regards,
Sujesh

**Pull request merged**